lib/schema.js

```diff
@@ -269,21 +285,43 @@ class Metaschema {
   buildCategory(category, ...args) {
     return this.categories.get(category).factory(...args);
   }
+
+  [initActions]() {
+    for (const [categoryName, { definition: category }] of this.categories) {
+      this.actions.set(
+        categoryName,
+        iter(Object.keys(category))
```

Comment:
Nit: I'd prefer to have this chain outside of `.set` call and have assigned a separate variable.